app/src/main/java/org/oppia/android/app/testing/LessonThumbnailImageViewTestFragment.kt

```diff
@@ -0,0 +1,21 @@
+package org.oppia.android.app.testing
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import org.oppia.android.app.fragment.InjectableFragment
+import org.oppia.android.databinding.LessonThumbnailImageViewTestFragmentBinding
+
+class LessonThumbnailImageViewTestFragment : InjectableFragment() {
```

Comment:
Ditto for this fragment (should have a KDoc).

Reply:
Done